gateway/src/main/java/io/camunda/zeebe/gateway/admin/backup/PartitionBackupStatus.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.admin.backup;
+
+import io.camunda.zeebe.protocol.impl.encoding.BackupStatusResponse;
+import io.camunda.zeebe.protocol.management.BackupStatusCode;
+import java.util.Optional;
+
+public record PartitionBackupStatus(
```

Comment:
:x: Please comment if this is truly public; but looking at its usages, it seems to be a package private thing, so we can also just downgrade the visibility (here and for its methods).